### PR TITLE
quiltctl: Tweaks and exec command

### PR DESCRIPTION
**Pull request by kklin**
- Fixes up some small things with quilctl
- Adds the `exec` command

### CODE REVIEW

**Comment by kklin** [comment]
Tacked on a small patch for speeding up unit tests.

**Comment by ejj** [comment]
FYI for the reviewer, merged the first couple of commits

**Comment by kklin** [comment]
Removed the `exec` commit because @mmussomele's work might make it cleaner; other stuff can still be reviewed.

**Comment by kklin** [comment]
`exec` commit is back :P

File: quiltctl/command/exec.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+	command         string
+
+	flags *flag.FlagSet
+}
+
+func (eCmd *Exec) createFlagSet() *flag.FlagSet {
+	flags := flag.NewFlagSet("exec", flag.ExitOnError)
+
+	flags.StringVar(&eCmd.host, "H", api.DefaultSocket,
+		"the host to query for machine information")
```
Comment (secant) [inline quiltctl/command/exec.go]: `container information`?

**Pull request closed without merging**